### PR TITLE
[6.12.z] leapp scenario 8.8 to 9.2

### PR DESCRIPTION
**Pull request by Satellite-QE**
Cherrypick of PR: https://github.com/SatelliteQE/robottelo/pull/11553

**PR Description**
- Upgrading Rhel host to next major RHEL Realse with job templates
- Path RHEL 8.8 to 9.2

**Steps:**
1. Import a subscription manifest and enable, sync source & target repositories
2. Create CV, add repositories, create lce, ak, etc.
3. Register contenthost using ak
4. Varify target rhel repositories are enable on Satellite
5. Update all packages, install leapp tool and fix inhibitors (in advance)
6. Run Leapp Preupgrade and Leapp Upgrade job template

**Expected output:**
- Update rhel major version to another major version

### CODE REVIEW

**Comment by Satellite-QE** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_leapp_client.py -k test_leapp_upgrade_rhel

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_leapp_client.py -k test_leapp_upgrade_rhel

File: testimony.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ CaseComponent:
     - katello-agent
     - katello-tracer
     - LDAP
-    - Leappintegration
+    - LeappIntegration
```
Comment (Gauravtalreja1) [inline testimony.yaml:70]: We need to rename this back, so either need to cherrypick this when https://github.com/SatelliteQE/robottelo/pull/12639 merges, or manually change it here

**Comment by vijaysawant** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_leapp_client.py -k test_leapp_upgrade_rhel

File: testimony.yaml (hunk 1 above)
Comment (vijaysawant) [reply testimony.yaml:70]: done

Comment (vijaysawant) [reply testimony.yaml:70]: @Gauravtalreja1 @jyejare @devendra104 
One check is failing due to component name issue

Comment (vijaysawant) [reply testimony.yaml]: sorry my bad, I haven't changed component name from `testimony.yaml`

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, PRT passed in run 4065 with commit ac0d42a42b75e5ea44538f16984242d054aa6751

**Pull request merged**